kernel/data-pipeline/distsql/handler/src/main/java/org/apache/shardingsphere/data/pipeline/migration/distsql/handler/update/MigrateTableExecutor.java

```diff
@@ -43,7 +43,7 @@ public final class MigrateTableExecutor implements DistSQLUpdateExecutor<Migrate
     @Override
     public void executeUpdate(final MigrateTableStatement sqlStatement, final ContextManager contextManager) {
         String modeType = contextManager.getInstanceContext().getModeConfiguration().getType();
-        ShardingSpherePreconditions.checkState(!"Standalone".equals(modeType), () -> new PipelineInvalidParameterException("Unsupported mode type of `Standalone`"));
+        ShardingSpherePreconditions.checkState("Cluster".equals(modeType), () -> new PipelineInvalidParameterException("Only `Cluster` of mode type is supported for now"));
```

Comment:
1, Could we invoke `InstanceContext.isCluster()`? Looks there's another one `Compatible_Cluster` except `Cluster`

2, Could we put current mode type in error message?